README.md

```diff
@@ -154,12 +160,10 @@ Then proceed to join a chat channel and send a message:
 ```lua
 -- send channel join message
 local channel_id = "pineapple-pizza-lovers-room"
-local channel_join_message = nakama.create_channel_join_message(1, channel_id, false, false)
-local result = nakama.socket_send(socket, channel_join_message)
+local result = socket.send_channel_join_message(socket, 1, channel_id, false, false)
 
 -- send channel messages
-local channel_message_send = nakama.create_channel_message_send_message(channel_id, "Pineapple doesn't belong on a pizza!")
-local result = nakama.socket_send(socket, channel_message_send)
+local result = socket.send_channel_message_send(channel_id, "Pineapple doesn't belong on a pizza!")
```

Comment:
It would be great if we could simplify these function names.

e.g. `socket.join_channel(...)` and `socket.send_channel_message(...)`

Reply:
Remember that this is generated code based on your API specs! For the socket and rest API functions I decided to name all functions sending message as "send_" + "message_as_snake_case".

The send_channel_message_send message is defined in your spec as ChannelMessageSend (https://github.com/heroiclabs/nakama-common/blob/master/rtapi/realtime.proto#L49) and the send_channel_join message comes form the ChannelJoin message (https://github.com/heroiclabs/nakama-common/blob/master/rtapi/realtime.proto#L41).

I guess we could skip the "send_" prefix? I'll do this!

Reply:
Yep, I think it works rather well without "send_" prefix on those functions